public/aboutus.html

```diff
@@ -217,6 +217,16 @@ <h3 class="text-xl font-extrabold mt-3 ">Pete Keleher</h3>
             </div>
           </div>
         </li>
+        <li class="mt-6 sm:mt-0">
+          <img src="../static/img/danielle-maxwell.png" alt="" class="rounded-t-xl">
+          <div class="bg-[#ECF6FF] rounded-b-xl">
+            <div class="px-6 pt-8 pb-5">
+              <h4 class="text-base font-bold text-[#1D65A6]">WebAssembly Specialist</h4>
+              <h3 class="text-xl font-extrabold mt-3 ">Danielle Maxwell</h3>
+              <p class="mt-3 text-base font-extralight">Junior Distributed Systems Engineer</p>
+            </div>
+          </div>
+        </li>
```

Comment:
Note: I also added this content to `index.html` because it wasn't clear to me which page is getting rendered in the site.